registry/format.md

```diff
@@ -5,18 +5,14 @@ permalink: /registry/format/index.html
 
 # Formats Registry
 
-## Master Issue
-
-* [#845](https://github.com/OAI/OpenAPI-Specification/issues/845)
-
 ## Contributing
 
 Please raise a [Pull-Request](https://github.com/OAI/OpenAPI-Specification/pulls) or [Issue](https://github.com/OAI/OpenAPI-Specification/issues) to contribute or discuss a registry value.
 
 ## Values
 
-|Value|Description|Issue|
-|---|---|---|
-{% for value in site.format %}| <a href="./{{ value.slug }}.html">{{ value.slug }}</a> | {{ value.description }} | {% if value.issue %}<a href="https://github.com/OAI/OpenAPI-Specification/issues/{{ value.issue }}">#{{ value.issue }}</a>{% endif %} |
+|Value|Description|Type|Source|Deprecated Since|
```

Comment:
```suggestion
|Value|Description|Type|Source|Deprecated|
```
I don't want to encourage people to tie deprecation to a specific version of something.  Ideally the value in this column is simply "true".  This is not intended as guidance to tell tooling vendors which formats to support for which versions of OpenAPI.  And I realize that would be very helpful to you.  However, that's a much bigger challenge and taking on a far bigger commitment.
I don't want people coming to us and saying "the OAI registry says this format is supported in this version of OpenAPI and therefore tool X should support it, why doesn't it".  Format support has always been the wild west.  I don't want us to retroactively make tools "non-compliant" to rules we are inventing after the fact.